contracts/contracts/harvest/FixedRateDripper.sol

```diff
@@ -58,7 +58,7 @@ contract FixedRateDripper is Dripper {
      * @param _perSecond Rate of WETH to drip per second
      */
     function setDripRate(uint192 _perSecond) external onlyGovernorOrStrategist {
-        emit DripRateUpdated(_perSecond, drip.perSecond);
+        emit DripRateUpdated(drip.perSecond, _perSecond);
```

Comment:
Did we mean to do this in this PR? Changes how the event is parsed.

If we were going to make a change, we could probably remove emitting the old rate.

Reply:
Yeah, I thought I'd fix this now and deploy it later. Removing old value makes sense, will do that

Reply:
On second thought, I just got rid of this change from this branch since it doesn't really belong here. Will make that change when we update the Dripper the next time